contracts/marketplace/Marketplace.sol

```diff
@@ -197,16 +230,30 @@ contract Marketplace is Ownable, Pausable, Destructible {
     );
 
     // Transfer asset owner
-    nonFungibleRegistry.safeTransferFrom(
+    nftRegistry.safeTransferFrom(
       seller,
       msg.sender,
       assetId
     );
 
 
-    bytes32 auctionId = auctionByAssetId[assetId].id;
-    delete auctionByAssetId[assetId];
+    bytes32 orderId = order.id;
+    delete orderByAssetId[nftAddress][assetId];
 
-    emit AuctionSuccessful(auctionId, assetId, seller, price, msg.sender);
+    emit OrderSuccessful(
+      orderId,
+      assetId,
+      seller,
+      nftAddress,
+      price,
+      msg.sender
+    );
   }
-}
\ No newline at end of file
+
+  function isContract(address addr) internal view returns (bool) {
```

Comment:
We already have this method as part of `ERC721Token` which is using  AddressUtils.

Reply:
👍

Reply:
I think none of the contracts we're inheriting from have that using